### ISSUE TITLE
Add label to select for screen readers

### ISSUE THREAD

**Issue by seejwy** [broadwayinc/skateui#76]
Screen readers are unable to understand the select field without a label property. Add one with visibility hidden.

**Comment by baksagimm** [comment]
labels are already implemented. if label props are given, it will generate the label element.
closing issue.
